### PR TITLE
List approved

### PR DESCRIPTION
**Pull request by henriquecalenzoo**
## Description

Created the component to list of approved and integrate with backend response

## Dev Notes

- If any developer notes are required, list them here. If not, remove the 'Notes' section.

### Steps

1. List all steps in a numbered list

## Screenshots / Videos

Add any relevant screenshots or videos. Any video should be added to the Jira ticket and linked here.

## Developer Checks

- [ ] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [ ] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

File: src/components/Approved/BannerApproved/index.tsx [hunk 1]
```diff
@@ -6,14 +6,10 @@ export const BannerApproved = () => (
...
-        A lista de aprovados para segunda fase já está disponível!
+        A lista de aprovados para o curso já está disponível!
       </S.BannerTitle>
-      <S.BannerDescription variant="body1">
-        Confira a lista e já fique por dentro das próximas etapas que estão por
-        vir!
-      </S.BannerDescription>
       <S.ContainerButton>
-        <LinkWrapper color="white" href="#Resultado">
+        <LinkWrapper color="white" href="#approved-list">
```
Comment (jonataspinto) [inline src/components/Approved/BannerApproved/index.tsx:12]: Acho que pode manter como resultado.

File: src/components/ListApprovedCourses/styles.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+})``;
+
+export const TitleApproved = styled(Typography).attrs({
+  variant: 'h2',
+})`
+  font-size: 3rem;
+  color: ${({ theme }) => theme.palette.design.purple};
+`;
+
+export const ListApprovedContainer = styled.div``;
```
Comment (jonataspinto) [inline src/components/ListApprovedCourses/styles.ts:87]: Estávamos ciando os containers com `section`. Agrega valor semantico.

File: src/components/ListApprovedCourses/styles.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    line-height: 2;
+  }
+`;
+
+export const Title = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
```
Comment (jonataspinto) [inline src/components/ListApprovedCourses/styles.ts:73]: Use rem 😊

Uma dica se não souber é só calcular o valor que vc quer em px, por nossa base para o rem que é 16px.

Ex: 

10 / 16 = 0.265 rem

File: src/components/ListApprovedCourses/styles.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
+import styled from 'styled-components';
+import { Typography } from '../Typography';
+
+type StyleProps = {
+  bgColor: 'green_dark' | 'blue_dark' | 'purple';
+  width?: string;
+};
+
+export const CourseContainer = styled.section`
```
Comment (jonataspinto) [inline src/components/ListApprovedCourses/styles.ts:9]: Aqui já não é section. A seção e o pai que agrupa todos esses.

**Review by jonataspinto — CHANGES_REQUESTED** [review]
Vamoss

File: src/components/Approved/BannerApproved/index.tsx (hunk 1 above)
Comment (jonataspinto) [reply src/components/Approved/BannerApproved/index.tsx:12]: portuga mesmo.

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? **GitHub runners could be the cause.**
[Try running on Foo instead](https://www.foo.software/docs/lighthouse-check-github-action/examples#running-on-foo-and-saving-results)

**Comment by estartandodevs** [comment]
**Size Change:** 0 B 🆕

**Total Size:** 0 B



<a href="https://github.com/preactjs/compressed-size-action"><sub>compressed-size-action</sub></a>

File: src/components/ListApprovedCourses/styles.ts (hunk 4 above)
Comment (henriquecalenzoo) [reply src/components/ListApprovedCourses/styles.ts:9]: Mas esse cara engloba todos.

Comment (henriquecalenzoo) [reply src/components/ListApprovedCourses/styles.ts]: Entendi o que quis dizer.

**Comment by estartandodevs** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.6 | ✅ |
| First Contentful Paint | 1.2 s |  |
| Time to Interactive | 6.9 s |  |
| Speed Index | 1.9 s |  |
| Total Blocking Time | 650 ms |  |
| Largest Contentful Paint | 5.1 s |  |
| Cumulative Layout Shift | 0.021 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.86 | ✅ |
| First Contentful Paint | 2.3 s |  |
| Time to Interactive | 6.9 s |  |
| Speed Index | 3.0 s |  |
| Total Blocking Time | 130 ms |  |
| Largest Contentful Paint | 2.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by ramonxm — APPROVED** [review]
LGTM

**Review by estartandodevs — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? **GitHub runners could be the cause.**
[Try running on Foo instead](https://www.foo.software/docs/lighthouse-check-github-action/examples#running-on-foo-and-saving-results)

**Comment by estartandodevs** [comment]
**Size Change:** 0 B 🆕

**Total Size:** 0 B



<a href="https://github.com/preactjs/compressed-size-action"><sub>compressed-size-action</sub></a>

**Pull request merged**